docs/root/configuration/http/http_filters/bandwidth_limit_filter.rst

```diff
@@ -0,0 +1,64 @@
+.. _config_http_filters_bandwidth_limit:
+
+Bandwidth limit
+====================
+
+* Bandwidth limiting :ref:`architecture overview <arch_overview_bandwidth_limit>`
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.bandwidth_limit.v3alpha.BandwidthLimit>`
+* This filter should be configured with the name ``envoy.filters.http.bandwidth_limit``.
+
+The HTTP Bandwidth limit filter limits the size of data flow to the max bandwidth set in the ``limit_kbps``
+when the request's route or virtual host has a per filter
+:ref:`bandwidth limit configuration <envoy_v3_api_msg_extensions.filters.http.bandwidth_limit.v3alpha.BandwidthLimit>`.
+
+If the bandwidth limit has been exhausted the filter stops further transfer until more bandwidth gets allocated
+according to the ``fill_interval`` (default is 50 milliseconds). If the connection buffer fills up with accumulated
+data then the source of data will be readDisabled as described in the :repo:`flow control doc<source/docs/flow_control.md>`.
```

Comment:
``readDisabled``

Reply:
while `readDisable(bool)` is code, 'readDisabled' is not. Do you still want me to mark it as a literal?

Reply:
hmm - not sure exactly - maybe rewording it a bit will help - reading this without prior knowledge might be confusing - perhaps something like - "...then the source of data will have `readDisable(True)` set as described..."

Reply:
Done. Hope that makes it clearer.